### ISSUE TITLE
#-1876683600------------- Buttons must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1116]
<strong>Source:</strong>`<button aria-label="" class="icon-end-empty" type="button"><calcite-icon class="icon icon--start" aria-hidden="true" icon="playF" scale="s" calcite-hydrated=""></calcite-icon></button>`

<strong>Element Location: </strong>["wmx-work-job-card:nth-child(33) > .c-jobCard > .c-jobCard__rightContainer--fixed.ng-star-inserted > .c-jobCard__actions > .c-jobCard__startBtn.t-startBtn[icon-start=\"playF\"]","button"]

<strong>Impact: </strong>critical

<strong>Failure Summary:</strong> Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,section508,section508.22.a,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/button-name?application=axeAPI
